ir/visitor.h

```diff
@@ -779,6 +781,8 @@ class Backtrack : public virtual Visitor {
     // returns true for passes that will never catch a trigger (backtrack() is always false)
 };
 
+std::ostream &operator<<(std::ostream &out, const Backtrack::trigger &trigger);
```

Comment:
not the `SplitFlowVisit_base`? It is in the .cpp.

Reply:
It turns out that `SplitFlowVisit_base::operator<<` usage is all inside visitor.cpp (in `void dump(const SplitFlowVisit_base *split)`), so I left it "private" for now.